composer.json

```diff
@@ -19,22 +19,23 @@
         "ext-openssl": "*",
         "guzzlehttp/guzzle": "^6.0",
         "monolog/monolog": "~1.11",
-        "sensio/framework-extra-bundle": "~3",
-        "surfnet/stepup-saml-bundle": "^4.0",
-        "symfony/config": ">=2.7,<4",
-        "symfony/dependency-injection": ">=2.7,<4",
-        "symfony/form": "~2.7|~2.8|~3,4",
-        "symfony/framework-bundle": ">=2.7,<4",
-        "symfony/http-kernel": ">=2.7,<4",
-        "symfony/twig-bridge": ">=2.7,<4",
-        "symfony/validator": ">=2.7,<4"
+        "sensio/framework-extra-bundle": "~3 | ^5.4",
+        "surfnet/stepup-saml-bundle": "^4.0 | ^4.1",
+        "symfony/config": ">=2.7,<4|^4.3",
+        "symfony/dependency-injection": ">=2.7,<4|^4.3",
+        "symfony/form": "~2.7|~2.8|~3,4|^3.4|^4.3",
+        "symfony/framework-bundle": ">=2.7,<4|^4.3",
```

Comment:
Strange we need Symfony completely for a small bundle.

Reply:
Also consider adding PHP 7.x as a Composer dependency. As this bundle will now only work with newer PHP versions